docs/core/docker/build-container.md

```diff
@@ -0,0 +1,391 @@
+---
+title: Containerize an app with Docker tutorial
+description: In this tutorial, you will learn how to containerize a .NET Core application with Docker.
+ms.date: 03/20/2019
+ms.topic: tutorial
+ms.custom: "mvc, seodec18"
+#Customer intent: As a developer, I want to containerize my .NET Core app so that I can deploy it to the cloud.
+---
+
+# Tutorial: Containerize a .NET Core app
+
+This tutorial teaches you how to build a Docker image that contains your .NET Core application. The image can be used to create containers for your local development environment, private cloud, or public cloud.
+
+In this tutorial you'll learn how to:
```

Comment:
My rant: I find starting the first two paragraphs with "this tutorial" repetitive and boilerplate. Consider:

```suggestion
You'll learn to:
```